kaitai/python/tests/utils.py

```diff
@@ -0,0 +1,205 @@
+# Utilities for tests cases for the python version of the Kaitai Struct SBP parser
+
+import kaitai.python.sbptable as kaitai_sbptable
+from kaitai.python.parse_utils import iter_messages as get_next_msg_kaitai, get_flattened_msg, serialise, BufferKaitaiStream
```

Comment:
As mentioned above, if we want to publish this to PyPI, import should look something like this:

```suggestion
from sbp.kaitai.parse_utils import iter_messages as get_next_msg_kaitai, get_flattened_msg, serialise, BufferKaitaiStream
```